---
title: '**Clearance of Clostridioides difficile colonization is associated with unique bacterial changes**'
csl: mbio.csl
fontsize: 12pt
output:
  word_document: default
  pdf_document:
    includes: null
    keep_tex: yes
  html_document: default
header-includes:
- \usepackage{setspace}
- \doublespacing
- \usepackage{lineno}
- \linenumbers
- \renewcommand{\familydefault}{\sfdefault}
- \usepackage{graphicx}
geometry: margin=1.0in
sansfont: Arial
bibliography: references.bib
---


```{r knitr_settings, eval=TRUE, echo=FALSE, cache=FALSE, warning=FALSE, message=FALSE}
library(knitr)
source('../code/functions.R')

opts_chunk$set("tidy" = TRUE)
opts_chunk$set("echo" = FALSE)
opts_chunk$set("eval" = TRUE)
opts_chunk$set("warning" = FALSE)
opts_chunk$set("cache" = FALSE)
opts_chunk$set("message" = FALSE)

```

```{r rdata_tables}
#metadata





```


\vspace{35mm}


Running title: Clearance of Clostridioides difficile colonization is associated with unique bacterial changes

\vspace{10mm}

Nicholas A. Lesniak${^1}$, Alyxandria M. Schubert${^2}$, Hamide Sinani${^3}$, Patrick D. Schloss${^1}$

\vspace{20mm}

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu

$1$ Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI 
$2$ FDA
$3$ ?????

\newpage
\linenumbers


## Abstract

The gut bacterial community plays an important role in preventing pathogens from colonizing the intestine. Many functions of the bacterial community preventing Clostridioides difficile from colonizing have been characterized, however we have not yet been able to successfully restore colonization resistance to a susceptible community. We observed clindamycin treated communities were able to be colonized by C. difficile and then spontaneously clear the colonization. Therefore, we investigated if other antibiotic-treated communities could be colonized and spontaneously clear the C. difficile colonization. We found intermediate dosages of antibiotics permitted the colonization and subsequent clearance of C. difficile. While all treatments resulted in colonization and clearance of C. difficile, we found each antibiotic had unique communities, bacterial changes and interactions with C. difficile. Generally, we found three different trends in susceptibility - transient Escherichia/Shigella bloom, repression of a few non-dominant bacteria, and sustained presence of diverse facultative anaerobes. We found clindamycin treatment led to a susceptibility associated with a relative bloom in the facultative anaerobe, Esherichia/Shigella. The clearance of C. difficile was concurrent with the reduction of Escherichia/Shigella and the restoration of the dominant bacteria. Whereas, streptomycin lead to relatively few changes to the community and there were only a few bacteria changing and interacting with C. difficile. Clearance in this scenario was associated with multiple Porphyromonadaceae OTUs. Lastly, cefoperazone led to a drastic reduction in the community structure resulting in a sustained increased abundance of many facultative anaerobes, allowing C. difficile to persist. Further elucidation of how C. difficile is colonizing different gut bacterial communities will help us to better identify and treat intestinal communities susceptible to C. difficile infection.

**Importance** The community of bacteria and other microbes in our intestines, our gut microbiome, provide us many benefits. One benefit of this bacterial community is preventing problematic bacteria from establishing itself in our intestine and causing an infection, this prevention is known as colonization resistance. Clostridioides difficile is especially susceptible to colonization resistance. C. difficile is unable to colonize when a healthy microbiome is present. However, when the gut community is modified by, for example, taking antibiotics, the bacteria affected by the antibiotic are no longer able to prevent colonization. This perturbation opens opportunities for C. difficile to colonize. C. difficile infections are primarily treated with antibiotics, however antibiotics don’t always work and can sometimes lead to recurrent infection. Therefore, we hope to understand how the bacterial community prevents and eliminates C. difficile colonization. We found that different bacterial communities colonized with C. difficile were able to spontaneously clear the colonization. Additionally, each treatment resulted in unique community changes with the clearance of C. difficile. These data indicate each community had different requirements for clearing the colonization. Understanding these clearance dynamics will better inform us the different ways C. difficile can persist and the microbiome can eliminate colonization.




\newpage

## Introduction

A complex consortium of bacteria and microbes inhabiting our gut prevent pathogens from colonizing and causing disease. This protection is known as colonization resistance. Bacteria prevent colonization through many mechanisms such as activating a host immune response, competing for nutrients, producing bacteriocins, and contributing to the maintenance of the mucosal barrier. Any perturbation of the intestinal community and these functions opens the possibility of a pathogen, like Clostridioides difficile, colonizing.

Once the gut community is perturbed, such as with antibiotic treatments, C. difficile is able to colonize and cause disease. C. difficile infection (CDI) especially problematic due to their burden on the healthcare system. Not only can C. difficile cause severe disease, such as toxic megacolon and death, but it is the primary cause of nosocomial diarrhea. CDI is primarily treated with antibiotics. The infections recalcitrant to antibiotics are eliminated by restoring the community with a fecal microbiota transplant (FMT), restoring the perturbed community to a healthier protective state. C. difficile spores are difficult to eliminate for the hospital environment, so we have a large population of susceptible people, antibiotic-treated patients, are in an environment in which C. difficile spores are abundant. Ideally, we need to identify susceptible patients and restore the colonization resistance of their gut microbiome.

In order to have the ability to identify susceptible microbial intestinal communities, we need to understand how the microbiome resists colonization. Studies have been working to elucidate the key microbes and mechanisms responsible for preventing C. difficile colonization and infection. Most studies thus far have looked for commonalities in colonized communities. However, we have not been able to successfully identify microbes that can be used as a therapeutic intervention to prevent or eliminate CDI. Previously, we have shown when C difficile colonizes different antibiotic-treated murine communities, it modifies its metabolism to fit each specific environment. Therefore, we hypothesize that each community has unique susceptibilities and requires specific changes to prevent and eliminate colonization. We made the observation that Clindamycin-treated communities spontaneously clear C. difficile colonization whereas mice treated with Streptomycin and Cefoperazone do not. We modified the level of the later two antibiotics to a level in which C. difficile was able to colonize at a high level ~10^7 and then be spontaneously cleared from the community. With these three different communities that all had the same starting community, colonized to a high level and spontaneously clear the colonization, we asked if the changes associated colonization and clearance we show each community had unique bacteria and community changes that were associated with clearance of C. difficile colonization. These data suggest that each case of CDI may need to be treated uniquely for the specific niche C. difficile has taken over.



## Results

**Clindamycin spontaneously clears C. difficile colonization**
Mice were treated with intraperitoneal injection of clindamycin (10 mg/kg) one day prior to challenge with C. difficile. Then each day for 10 days, stool was collected for enumerating C. difficile CFUs and sequencing the 16S rRNA genes of the community. All mice were colonized to a high level and cleared the colonization within 10 days (Figure 1A). We then looked at the 16S rRNA sequences to understand the community changes occuring while C. difficile colonization is being reduced. The diversity of the community decreases greatly with antibiotic treatment and does not recover within the time period that C. difficile colonization is reduced. However, the changes of the colonized community are similar to the changes observed in a mock challenged clindamycin-treated community. Generally, we observe at the time of C. difficile challenge the community is dominated by Escherichia, which decreases over the following six days, concurrent with reduction in C. difficile CFU level. We see an opposite trend in Porphyromonadaceae and Bacteroides which are greatly reduced with antibiotics but recover as C.difficile colonization is reduced.

**C. difficile is spontaneously cleared when reducing Cefoperazone and Streptomycin concentrations**
We first ran treated mice with Streptomycin and Cefoperazone at concentrations of typical C. difficile models for 10 days to observe long-term colonization dynamics. In both experiments, C. difficile colonization was maintained (Figure 2A). Next, we repeated the C. difficile challenge with a two-step reduction in antibiotic concentration. In both conditions we observed the lowest concentration we either lowly and transiently colonized or not colonized (Figure 2C). Whereas the intermediate concentration of both antibiotics resulted in a high level colonization in most mice and resulted in a mix of communities that cleared the colonization while the other communities allowed the colonization to persist (Figure 2B). We next compared the 16S rRNA gene sequences of the communities at the time of infection. Cefoperazone treatment shifted the dominant community members from a community dominated by Porphyromonadaceae, Bacteroides and Akkermansia to one dominated by Lactobacillus and Escherichia with the increased concentrations (Figure 2D). Streptomycin treatment maintained the dominant community genera Porphyromonadaceae, Bacteroides and Akkermansia but reduced most of the other genera of the community (Figure 2E).

**Bacteria abundance changes associated with C. difficile colonization are unique to each treatment**
With these three different treatments of the same naive gut community, we investigated if the changes associated with clearance of C. difficile colonization we conserved. To begin, we compared the beta-diversity of the major time points to determine if the communities were similar (Figure 3A). The clindamycin-treated communities had a large shift with antibiotic treatment and only slight recovery of diversity by the time C. difficile was no longer colonized. Cefoperazone-treated communities mostly experienced a large shift in diversity with antibiotic treatment but were divergent in the recovery. Communities no longer colonized with C. difficile had shifted back to be most similar to their initial community structure, whereas the communities with C. difficile still present had little change in community structure from post-antibiotic treatment. Streptomycin-treatment communities exhibited no community structural change differences between ones that remained colonized compared to those that had the colonization eliminated. These communities experienced a slight shift away from their initial community with antibiotic treatment and all returned to a community structure closely related to their initial community. We also compared how different each community was from others in the same treatment as well as to the other treatments. There was a wide range of variation in dissimilarity of community structures when comparing across mice, with most communities being more similar to mice within treatment than across treatments, except cefoperazone. Cefoperazone-treated communities were near equally dissimilar within treatment as across treatment. Additionally, those cefoperazone-treated communities persistently colonized with C. difficile were more dissimilar at the end of the experiment. Overall, these data suggest returning to the initial community is important for cefoperazone, but does not appear to have an effect on the other antibiotics.

Since these antibiotic-treated communities are different from each other and have different relationships between the bacterial changes and colonization elimination we investigated what changes occur in each condition. First, we compared the difference in abundance of bacterial operational taxonomic units (OTUs) between communities that ended with or without C. difficile present (Figure 3B). At the time of infection, cefoperazone-treated communities that remained colonized were significantly enriched in facultative anaerobes, Streptococcus, Lactobacillus, Enterococcus, Pseudomonas, Staphylococcus and Escherichia. Whereas, cefoperazone-treated communities that had C. difficile colonization cleared had significant enrichment in eight different OTUs belonging to Porphyromonadaceae at the time of clearance. Streptomycin-treated communities had fewer differences between outcomes. Communities that remained colonized were depleted of Alistipes at time of infection and clearance and depleted of two Porphyromonadaceae OTUs. Next, we determined the significant changes occurring between major time points (Figure 3C). Clindamycin-treated communities show large decreases in 23 OTUs and a nearly complete dominant bloom of Escherichia. With the elimination of C. difficile, we observed a drastic reduction of Escherichia and recovery of 9 OTUs belonging to Porphyromonadaceae, Bacteroides, Akkermansia, Lactobacillus, Bifidobacterium and Lachnospiraceae. Cefoperazone-treated community differences primarily observed in antibiotic treatment in communities where C. difficile persisted. These communities had a loss of numerous Porphyromonadaceae OTUs and increases in the facultative anaerobes differentiating the cefoperazone-treated communities with different outcomes (Figure 3B). Lastly, Streptomycin-treated community temporal changes were more subtle. Most of the differences were observed with antibiotic treatment and many of those differences were recovered by the end of the experiment. Generally, the communities that were no longer colonized had more increased abundance of Porphyromonadaceae OTUs and fewer lost OTUs at the time of infections. At the end of the experiment the temporal differences observed in the communities without C. difficile were various OTUs reducing or increasing to around 1% relative abundance, whereas in communities with C. difficile present had the depletion of Olsenella and increase of three Porphyromonadaceae OTUs to nearly 10% relative abundance. Together, these data reveal antibiotic-treatment specific dynamics associated with the elimination of C. difficile colonization.

**Distinct features of the bacterial community at the time of infection can classify endpoint colonization**
With observation of the distinct community differences within each treatment, we sought to classify whether colonization would be eliminated with the snapshot of the community at the time of C. difficile challenge. We employed a machine learning model, L2 Logistic Regression, to determine the OTUs that contributed most to differentiating the colonization outcome (Figure 4). Our model was able to correctly classify our samples with an AUROC of 0.9XX. We next randomly permuted each OTU feature to determine its contribution to correctly classify the outcome. This analysis revealed many of the OTUs in the previous analyses with the added information that in the context of the other microbes and treatments the model is able to correctly classify the outcome. Comparing the abundance of the most impactful OTUs, we can see where the community context matters. For example, the OTU with the largest effect on the model, Escherichia, in cefoperazone high levels signals persistence, whereas in clindamycin a high level leads to clearance. Therefore, the model is relying on contributions from multiple OTUs to classify the experimental outcome. These data can offer a basis for hypothesizing about what combination of bacteria promote the ability to eliminate C. difficile colonization.
    
**Conditional independence networks reveal treatment-specific relationships between the community and C.difficile during colonization clearance**
While we can hypothesize about which community members are most important to each treatment at the time of infection to correctly classify the outcome, we inquired how the changes in C. difficile colonization levels related to the changes in bacterial abundance to elucidate the bacteria directly affecting the colonization level of C. difficile. We developed a conditional independence network for each treatment using SPIEC-EASI (sparse inverse covariance estimation for ecological association inference) to analyze the temporal abundance data and initially focused on the first-order connections to C. difficile (Figure 5A). We observed C. difficile has limited interactions in cefoperazone-treated communities, the primary relationship being a positive with Escherichia. In clindamycin-treated communities, C. difficile colonization has positive relationships with Escherichia, Pseudomonas, and Olsenella and negative relationships with Lachnospiraceae and Clostridium XIVa. In streptomycin-treated communities, C. difficile has negative relationships with Porphyromonadaceae and positive relationships with Ruminococcaceae, Bacteroidetes, Clostridium IV and Olsenella. Finally, we also compared the whole network (Figure 5B). This analysis revealed cefoperazone treatment results in a network with much lower degree centrality, thus these communities as a whole are composed of primarily single connections. Additionally, cefoperazone has significantly lower betweenness centrality and those communities which C. difficile have even lower betweenness centrality, therefore cefoperazone-treated communities are depleted in bacteria that connect with many other bacteria.



## Discussion 

Previous studies investigating how gut microbial communities resist C. difficile colonization have sought to identify common microbes of resistant communities or communities that have cleared colonization. Here, we elucidated community-specific changes co-occuring with the clearance of C. difficile colonization. Even though these communities all began with closely related communities, colonization clearance occurred with treatment-specific changes and not necessarily returning to their baseline community. These data suggest that the same treatment to restore colonization resistance may not be effective in all cases. A specific treatment designed for the specific susceptibilities will be most effective to prevent and eliminate C. difficile colonization. Further investigation into specific cases of C. difficile colonization will elucidate the niches C. difficile utilizes and the interventions necessary to prevent C. difficile from colonizing or eliminating C. difficile colonization.

In this study we observed three scenarios in which the microbiome is able to spontaneously eliminate C. difficile colonization. We can categorize the general changes, microbial relationships and potential interventions of these colonization scenarios. (1) Right place, right time - a transient increase in facultative anaerobes permits C. difficile colonization but quickly recovers and C. difficile is not able to persist. Since this susceptibility is due to a transient decrease in the community, interventions could possibly make the infection worse. If time is sufficient for the community to recover and eliminate infection, but the community was treated with an antibiotic or the preparation for an FMT, this may allow C. difficile to spread to other niches and prolong the infection. This susceptibility may be identified by decreasing abundances of facultative anaerobes. (2) Single meal option menu, take it or leave it - a perturbation only affects a select portion of the microbiome, leading to minor abundance changes and a slight decrease in diversity, opening a limited specific niche for C. difficile to colonize. A few specific bacteria would be necessary to recolonize the exposed niche and eliminate C. difficile colonization. A fecal microbiota transplant or potentially a probiotic with high diversity may seed the necessary bacteria to clear C. difficile from the exposed niche of the colonized community. (3) All day all-you-can-eat buffet - a significant perturbation occurs leading to a persistent increase in facultative anaerobes and many different niches become available. C. difficile is able to colonize numerous niches. A full FMT may be sufficient but this scenario is sensitive to there being sufficient overlap between the exposed niches and the diversity of the transplanted community. It may be necessary to do multiple FMTs or using an enriched functionally-diverse fecal community transplant to recover multiple niches and replace the lost functional redundancy to clear C. difficile colonization.

Based on these observations, facultative anaerobes could be one potential way to detect a community may be susceptible to C. difficile colonization. In our data, C. difficile colonized when facultative anaerobes increased in abundance. The abundance increase could be related to the loss of oxygen level suppression by obligate anaerobes allowing the expansion of facultative anaerobes. Future studies should further investigate this relationship between facultative anaerobes and susceptibility to identify markers of aerobic conditions and interventions to assist the community to recover those changes to prevent colonization.

Our current method of treating recurrent CDI is to give infected patients fecal stool form a healthy is highly effective at eliminating CDI, however it does not work in all patients. The findings in this study may help explain why FMTs may be ineffective. While the FMT is transplanting a whole community, some members and functions will be lost through the aerobic preparation and transfer process, potentially bacteria necessary to fill a subset of patients susceptibilities. Therefore, understanding how a community is susceptible and how C. difficile is colonizing the gut microbiome will better inform treatments.



## Materials and Methods

**Animal care**
All mice were obtained from a single breeding colony and maintained in specific-pathogen-free (SPF) conditions at the University of Michigan animal facility. All mouse protocols and experiments were approved by the University Committee on Use and Care of Animals at the University of Michigan and completed in agreement with approved guidelines.

**Antibiotic administration**
Mice were given one of three antibiotics, cefoperazone, clindamycin, or streptomycin. Cefoperazone (0.5, 0.3, or 0.1 mg/ml) and streptomycin (5, 0.5, or 0.1 mg/ml) were delivered via drinking water for 5 days. Clindamycin (10 mg/kg) was administered through intraperitoneal injection. 

**C. difficile challenge**
Mice were returned to untreated drinking water for 24h before challenging with C. difficile strain 630Δerm spores. C. difficile spores were aliquoted from a single spore stock stored at 4℃. Spore concentration was determined one week prior to the day of challenge (). 103 C. difficile spores were orally gavaged into each mouse. Once the gavages were completed, the remaining spore solution was serial diluted and plated to confirm accurate spore concentration was delivered.

**Sample collection**
Fecal samples were collected on the day antibiotic treatment was started, on the day of C. difficile challenge and the following 10 days. For the day of challenge and beyond, a fecal sample was also collected and  weighed. Then under anaerobic conditions a fecal sample was serial diluted in anaerobic phosphate-buffered saline and plated on TCCFA plates. After 24-h of anaerobic incubation at 37℃ CFU were enumerated ().

**DNA sequencing**
Total bacterial DNA was extracted from each fecal sample using MOBIO PowerSoil-htp 96-well soil DNA isolation kit. We created amplicons of the 16s rRNA gene V4 region and sequenced them using an Illumina MiSeq as described previously ().

**Sequence curation**
Sequences were processed using mothur(v.1.XX) as previously described (). Briefly, we used a 3% dissimilarity cutoff to group sequences into operational taxonomic units (OTUs). We used a naive Bayesian classifier with the Ribosomal Database Project training set (version XX) to assign taxonomic classifications to each OTU (). With the fecal samples, we also sequenced a mock community with a known community composition and their true 16s rRNA gene sequences. We processed this mock community along with our samples to determine our sequence curation resulted in an error rate of X.XX%

**Statistical analysis and modeling**
Diversity comparisons were calculated in mothur. To compare within community diversty, we calculated Sobs and Inverse Simpson. To compare across communities, we calculated dissimilarity matrices based on metric of Yue and Clayton (). Samples were rarified to 2,000 sequences per sample within calculating the distance matrices to ensure no biases remained due to uneven sampling.  OTUs were subsampled to 2,000 counts per sample and remaining statistical analysis and data visualization was performed in R (v#.#.#) with the tidyverse package (v#.#.#). Pairwise OTU-to-OTU comparisons were calculated by pairwise Wilcoxon rank sum test and then P values were corrected for multiple comparisons with a Benjamini and Hochberg adjustment for a type I error rate of 0.05 (). Logistic regression models were contructed using half the data to train and the other to test the model.  The model was developed from the caret R package and previously developed machine learning pipeline (,). Conditional independence networks were calculated using  SPIEC-EASI (sparse inverse covariance estimation for ecological association inference) methods from the SpiecEasi R package after optimizing lambda to 0.001 with a network stability between 0.045 and 0.05 (). Network centrality measures degree and betweenness were calculated on whole networks using functions from the igraph R package ().

**Code availability**
Scripts necessary to reproduce our analysis and this paper are available in an online repository (https://github.com/SchlossLab/Lesniak_Clearance_XXXX_2020).

**Sequence data accession number**
    All 16s rRNA gene sequence data and associated metadata are available through the Sequence Read Archive via accession no. SXX#######.

## Acknowledgements 

Thank you to Begum Topcuoglu and Sarah Tomkovich for critical discussion in the development and execution of this project.
This work was supported by several grants from the National Institutes for Health R01GM099514, R01HG005975, U19AI090871, and P30DK034933. Additionally, NAL was supported by the Molecular Mechanisms of Microbial Pathogenesis training grant (NIH T32 AI007528).
The funding agencies had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.


\newpage

![Figure 1](../results/figures/figure_1.jpg)

**Figure 1. Clindamycin spontaneously clears C. difficile colonization** (A) Daily CFU of C. difficile in fecal samples of clindamycin-treated mice from time of challenge (Day 0) through 10 days post infection (dpi). The bold line is the median CFU of the group and the transparent lines are the individual mice.  (B) Quantification of diversity over time from pre-treatment (Day -1) through 10 dpi. Red colored boxes were challenged with C. difficile 630, whereas uncolored gray boxes were mock-challenged. (C) Daily relative abundance of ten most abundant genera, all other genera grouped into Other. Color intensity is log10-transformed mean percent relative abundance of each day. (N = 11 mice). Limit of detection (LOD). Boxplot shows median - horizontal bar, quartile range - box, range of non-outlier data - whiskers, data greater than 1.5 times the interquartile range - points.


![Figure 2](../results/figures/figure_2.jpg)

**Figure 2. C. difficile is spontaneously cleared when reducing cefoperazone and streptomycin concentrations** (A) Daily CFU of C. difficile in fecal samples of cefoperazone and streptomycin-treated mice from time of challenge (Day 0) through 10 days post infection (dpi). Antibiotic concentrations used were C. difficile infection model levels (0.5 mg/mL and 5.0 mg/mL). The bold line is the median CFU of the group and the transparent lines are the individual mice. (B) Daily CFU of C. difficile in fecal samples of cefoperazone and streptomycin-treated mice from time of challenge (Day 0) through 10 days post infection (dpi). Antibiotic concentrations used were two-step decreases from C. difficile infection model levels (0.3 and 0.1 mg/mL and 0.5 and 0.1 mg/mL, respectively). The bold line is the median CFU of the group and the transparent lines are the individual mice. (C) Relative abundance of twelve most abundant genera on the day of C. difficile challenge (Day 0), all other genera grouped into Other. Color intensity is log10-transformed mean percent relative abundance of each day. (cefoperazone - 0.5 mg/mL N = 5, 0.3 mg/mL N = 9, 0.1 mg/mL N = 2; streptomycin - 5.0 mg/mL N = 8, 0.5 mg/mL N = 7, 0.1 mg/mL N = 7). Limit of detection (LOD).


![Figure 3](../results/figures/figure_3.jpg)

**Figure 3. Bacteria abundance changes associated with C. difficile colonization are unique to each treatment** (A) Beta diversity comparisons - Initial vs Initial compares one pre-antibiotic treatment to all other pre-antibiotic treatment samples; Initial vs Time of Infection (TOI) and End vs Initial compare time points within mouse, intra-comparisons compare a sample to all other samples receiving the same antibiotic treatment and inter-comparisons compares a sample to all other samples not receiving the same antibiotic treatment mouse to all other. Boxes colored by time point being compared to the same time point or to the initial time point. Comparisons restricted to within antibiotic and outcome except inter-comparisons. (B) The difference in the relative abundance of OTUs that are significantly different between communities that were no longer colonized by C. difficile and those that remained samples colonized within each antibiotic treatment for each timepoint. (C) The difference in the relative abundance of OTUs that are significantly different communities between time points within each C. difficile colonization outcome for each antibiotic treatment. Bold OTUs were shared across comparisons. Significant differences in panel B and C were calculated by Wilcoxon rank sum test with Benjamini-Hochberg correction. Limit of detection (LOD). Boxplot shows median - horizontal bar, quartile range - box, range of non-outlier data - whiskers, data greater than 1.5 times the interquartile range - points.


![Figure 4](../results/figures/figure_4.jpg)

**Distinct features of the bacterial community at the time of infection can classify endpoint colonization** (A) Classification performance of L2 logistic regression. Area under the receiver-operator curve for classifying if the community will remain colonized based on the OTUs present at the time of C. difficile challenge (Day 0). (B) Feature importances determined by the negative effect randomizing an individual feature has on the model performance. All OTUs affecting performance shown. (C) Relative abundance difference in features used by L2 logistic regression model faceted by antibiotic treatment. Red boxplot - antibiotic-treated communities that were no longer colonized; Blue boxplot - antibiotic-treated communities that remained colonized. Boxplot shows median - horizontal bar, quartile range - box, range of non-outlier data - whiskers, data greater than 1.5 times the interquartile range - points.



![Figure 5](../results/figures/figure_5.jpg)
**Figure 5. Conditional independence networks reveal treatment-specific relationships between the community and C.difficile during colonization clearance** (A) SPIEC-EASI (sparse inverse covariance estimation for ecological association inference) networks showing conditionally independent first-order relationships between C. difficile and the community as it is cleared for the gut environment. Nodes sized by median relative abundance of OTU. Edges thickness is determined by the relationship strength and red for negative interactions and blue for positive interactions. (B) Network centrality measured with betweenness, how many paths between two OTUs pass through an individual, and degree, how many connections an OTU has. Colored by antibiotic (cefoperazone - blue, clindamycin - red, streptomycin - yellow) and boxplots are filled in gray for networks that remained colonized. Boxplot shows median - horizontal bar, quartile range - box, range of non-outlier data - whiskers, data greater than 1.5 times the interquartile range - points.



\newpage

## References